### DISCUSSION TITLE
TypeError with some libs - Can we upgrade Vite's version?

### DISCUSSION THREAD

**Discussion by suleimanas** [quasarframework/quasar#16617] [Ideas / Proposals]
Hello, I have an app that uses @quasar/app-vite, and some of the libraries like surrealdb doesn't work with it. i get this error:
"TypeError: import_ws.WebSocket is not a constructor" - i could swear that it was working before

if I go `\node_modules\@quasar\app-vite>` then upgrade vite `npm i vite@latest` it'll be working fine
so could we upgrade Vite's version in the next release :) , I don't want to open an issue before checking with you.. thanks

**Comment by yusufkandemir** [comment]
`@quasar/app-vite` v2 will use Vite v5: https://github.com/quasarframework/quasar/issues/14077
Also, you can already upgrade the Vite version as a workaround, see the related issue.
